### PR TITLE
Drop ruby 2.4 support/ Config Github Actions

### PR DESCRIPTION
**Pull request by joaocv3**
Hi @thibaudgg  :)

The new actions cannot be run directly here yet.

 So you can check those changes working here: https://github.com/joaocv3/video_info/pull/2 (it's the same branch)

What do you think?

### CODE REVIEW

File: video_info.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.test_files   = s.files.grep(%r{^spec/})
   s.require_path = 'lib'
 
-  s.required_ruby_version = '>= 2.2.0'
+  s.required_ruby_version = '> 2.4.0'
```
Comment (thibaudgg) [inline video_info.gemspec:21]: What about that instead? I find it easier to parse.

```suggestion
  s.required_ruby_version = '>= 2.5.0'
```

**Review by thibaudgg — COMMENTED** [review]
Looks great, I'm all in to switch away from Travis and use GitHub actions. :+1:

Maybe just update the `required_ruby_version` and we should be good, thanks!

File: video_info.gemspec (hunk 1 above)
Comment (joaocv3) [reply video_info.gemspec]: Sure! Looks better

**Comment by thibaudgg** [comment]
@joaocv3 one point that I missed, could you please also update the build status page in the README, Github provides one as well. 

``` Ruby
[![Ruby Unit Tests Action Status](https://github.com/thibaudgg/video_info/workflows/Ruby%20Unit%20Tests/badge.svg)](https://github.com/thibaudgg/video_info/actions?query=workflow%3A%22Ruby+Unit+Tests%22)
```

**Pull request merged**